cmd/touch/touch.go

```diff
@@ -0,0 +1,69 @@
+package touch
+
+import (
+	"bytes"
+	"time"
+
+	"github.com/ncw/rclone/cmd"
+	"github.com/ncw/rclone/fs"
+	"github.com/spf13/cobra"
+)
+
+var (
+	notCreateNewFile bool
+	timeAsArgument   string
+)
+
+func init() {
+	cmd.Root.AddCommand(commandDefintion)
+	flags := commandDefintion.Flags()
+	flags.BoolVarP(&notCreateNewFile, "not-create", "C", false, "Do not create the file if it does not exist.")
+	flags.StringVarP(&timeAsArgument, "time", "t", "", "Change the modification times to the specified time instead of the current time of day. The argument is of the form 'YYMMDD' (ex. 17.10.30) or 'YYYY-MM-DDTHH:MM:SS' (ex. 2006-01-02T15:04:05)")
```

Comment:
There is already a `--time` on touch, perhaps this should be `timestamp` instead?

Reply:
Oh yes, my bad. I only read man touch in terminal (mac OS), which has less information than linux version. 
Rclone doesn't support to change access time?

Reply:
No it only supports the modification time